.travis.yml

```diff
@@ -79,6 +79,10 @@ script:
       presto-product-tests/bin/run_on_docker.sh \
           multinode-tls-ldap -g smoke,cli,group-by,join,tls
     fi
+    if [[ -v PRODUCT_TESTS_SPECIFIC_ENVIRONMENT ]]; then
+      presto-product-tests/bin/run_on_docker.sh \
```

Comment:
I'm trying to get TLS support landed in the very, very near future. David has been reviewing it and I've been addressing his feedback.